NEWS.md

```diff
@@ -21,6 +21,11 @@
   - `$scan_parquet()` gains an argument `hive_partitioning`.
   - `$meta$tree_format()` has a better formatted output.
 
+## What's changed
+
+- New class `RPolarsSQLContext` and its methods.
```

Comment:
Can you add some information on what it allows to do? Something like: 
```suggestion
- New class `RPolarsSQLContext` and its methods to perform any SQL query on a Polars DataFrame or LazyFrame.
```
(not sure what I added is correct)